src/modules/sdk/objections/types.ts

```diff
@@ -37,6 +37,8 @@ export interface ObjectionCreate {
  * @interface
  */
 export interface ObjectionPatch {
+  fullName?: string;
```

Comment:
json fields should be snake_case

Reply:
I agree. There are a number of other fields in camel case, so what I'd like to do is raise a bug for this and defer to that as there is other work depending on these fields at present and it will need to be tersted thoroughly.